content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,80 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl using package manager
```

Comment:
A minor adjustment to the heading by using `Uninstall kubectl` instead. This change will ensure that the heading accurately reflects the content, which includes the removal of kubectl installation via curl too.

```suggestion
### Uninstall kubectl
```